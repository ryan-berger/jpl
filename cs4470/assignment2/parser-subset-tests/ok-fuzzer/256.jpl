read image "" to Z2i_
assert G6p.(), ""
write image S(8, t(q4v_, 02.937, a, h, .442, 0), x, .2, s, 3) to "p"
assert j, "Zf"
time time read image "" to D3w_
time time let E6O.9C_5F_7s.6y_3i_ = A1S_4r_3Z_5Y_6g_4P.8K_7B.()
assert C(E()), ""
write image 3 to "N"
write image .419 to ""
print "&"
time read image "" to z
