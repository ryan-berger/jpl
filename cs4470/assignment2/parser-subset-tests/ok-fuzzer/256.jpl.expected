(ReadImageCmd "" (VarArgument Z2i_))
(StmtCmd (AssertStmt (CallExpr G6p.) ""))
(WriteImageCmd
 (CallExpr
  S
  (IntExpr 8)
  (CallExpr
   t
   (VarExpr q4v_)
   (FloatExpr 2)
   (VarExpr a)
   (VarExpr h)
   (FloatExpr 0)
   (IntExpr 0))
  (VarExpr x)
  (FloatExpr 0)
  (VarExpr s)
  (IntExpr 3))
 "p")
(StmtCmd (AssertStmt (VarExpr j) "Zf"))
(TimeCmd (TimeCmd (ReadImageCmd "" (VarArgument D3w_))))
(TimeCmd
 (TimeCmd
  (StmtCmd
   (LetStmt
    (ArgLValue (VarArgument E6O.9C_5F_7s.6y_3i_))
    (CallExpr A1S_4r_3Z_5Y_6g_4P.8K_7B.)))))
(StmtCmd (AssertStmt (CallExpr C (CallExpr E)) ""))
(WriteImageCmd (IntExpr 3) "N")
(WriteImageCmd (FloatExpr 0) "")
(PrintCmd "&")
(TimeCmd (ReadImageCmd "" (VarArgument z)))
